domain/machine/service/service.go

```diff
@@ -16,6 +16,19 @@ type State interface {
 
 	// DeleteMachine deletes the input machine entity.
 	DeleteMachine(context.Context, string) error
+
+	// AllMachines retrieves the ids of all machines in the model.
+	// If there's no machine, it returns an empty slice.
+	AllMachines(context.Context) ([]string, error)
```

Comment:
Why have `ListMachines` vs `AllMachines`?

Reply:
yeah I wanted to rename it but looks like I only did in the state👍